src/app/app.component.ts

```diff
@@ -101,18 +103,20 @@ export class AppComponent {
       from(getFromStore<YoutubeFeatureState>(this.settingsLocation, 'youtube')),
     ])
       .pipe(takeUntilDestroyed())
-      .subscribe(([
-        config,
-        openai,
-        twitch,
-        azure,
-        elevenLabs,
-        vtubeStudio,
-        vstream,
-        appSettings,
-        watchStreak,
-        youtube,
-      ]) => {
+      .subscribe((
```

Comment:
Unrelated but you gotta move off combineLatest arrays, I find positionally-driven mappings like this to be super error-prone. Switch to `combineLatest({ name: obs$, name2: obs2$ })`

Reply:
Yeah I really should.